### PR TITLE
Add laravel installer page

### PR DESCRIPTION
**Pull request by teguh02**
I add laravel installer page like a wordpress installer page to make it easier for everyone

### CODE REVIEW

**Comment by nasirkhan** [comment]
Thank you for submitting the pull request. 

It would be a nice to have feature but I have a few opinion about this. 

1. To build the installer feature, you added rashidlaasri/LaravelInstaller package. I checked this package and the last release was more than a year ago. There are a number of open issues which is from from January 2019, last closed issue was on January 2021. So I think this is not an active project. What if the maintainer declare the project as archived? 
2. You can compare the Articles module with the Wordpress posts feature, but this is a started project and the target users are not large as Wordpress. You may build your application on top of this, which will save a big chunk of your time.  
3. The installer have a nice feature of checking the dependencies before installing. You can make a command to have this feature.
4. One goal of this project is to be simple and be less dependent on other packages. So that it will not became a road block the main product.  

In summary, I would like to use such packages which updates regularly. It is a starter project so it should have only the most useful features. Additional features can be added as separate modules. So I am sorry that I can not accept this pull request. 

I am very happy that you are interested in improving this project. I would like to request you to check the list at https://github.com/nasirkhan/laravel-starter/issues/269, you may work on the features listed there or suggest something new as well.
